shared/src/commonMain/kotlin/com/vml/tutorial/plantshop/profilePreferences/presentation/editAddress/components/EditAddressScreen.kt

```diff
@@ -0,0 +1,164 @@
+package com.vml.tutorial.plantshop.profilePreferences.presentation.editAddress.components
+
+import androidx.compose.foundation.layout.Arrangement
+import androidx.compose.foundation.layout.Column
+import androidx.compose.foundation.layout.Spacer
+import androidx.compose.foundation.layout.fillMaxWidth
+import androidx.compose.foundation.layout.height
+import androidx.compose.foundation.layout.padding
+import androidx.compose.foundation.layout.size
+import androidx.compose.foundation.lazy.LazyColumn
+import androidx.compose.foundation.rememberScrollState
+import androidx.compose.foundation.text.KeyboardOptions
+import androidx.compose.foundation.verticalScroll
+import androidx.compose.material.icons.Icons
+import androidx.compose.material.icons.rounded.ArrowBack
+import androidx.compose.material3.Button
+import androidx.compose.material3.CircularProgressIndicator
+import androidx.compose.material3.Icon
+import androidx.compose.material3.IconButton
+import androidx.compose.material3.MaterialTheme
+import androidx.compose.material3.Text
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.graphics.Color
+import androidx.compose.ui.text.input.ImeAction
+import androidx.compose.ui.text.input.KeyboardType
+import androidx.compose.ui.text.style.TextAlign
+import androidx.compose.ui.unit.dp
+import com.vml.tutorial.plantshop.MR
+import com.vml.tutorial.plantshop.core.presentation.UiText
+import com.vml.tutorial.plantshop.core.presentation.asString
+import com.vml.tutorial.plantshop.profilePreferences.presentation.ScreenTitle
+import com.vml.tutorial.plantshop.profilePreferences.presentation.UserInput
+import com.vml.tutorial.plantshop.profilePreferences.presentation.editAddress.EditAddressState
+import com.vml.tutorial.plantshop.ui.theme.Typography
+
+@Composable
+fun EditAddressScreen(state: EditAddressState, onEvent: (EditAddressEvent) -> Unit) {
+    LazyColumn {
+        item {
+            IconButton(
```

Comment:
I think the back button should stick to the top, and not move with the content

Reply:
I have an idea, but you can ignore me and leave it as it is now and we can get back later, it don't have to be perfect :) So my idea ->
If you scroll up the title goes up -> next to the back button and the "app bar"'s row gets a background color. Usually, this is how they solve this kind of UI issue. Idk how hard is to implement it in compose but you can try it if you like.

Reply:
Now, the back button is fixed at the top. I like your suggestion, but I believe it would be better to do it in a separate PR. This is because I am planning some UI changes for the other screens as well